docs/widgets/rule.md

```diff
@@ -68,3 +68,8 @@ This widget sends no messages.
 ::: textual.widgets.Rule
     options:
       heading_level: 2
+
+::: textual.widgets.rule
+    options:
+      show_root_heading: true
+      show_root_toc_entry: true
```

Comment:
Doesn't this make it so that the widget itself shows up twice? (I'm not sure, I'm doing some guesswork here.)

Reply:
Ahh of course, you added the `rule.py` with the reimports. Thanks a ton for the screenshot :)